Makefile

```diff
@@ -66,10 +68,20 @@ test-clover:
 	docker build -f examples/clover/Dockerfile .
 
 publish:
-	$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-*"
+	$(AWS) s3 cp \
+	  --acl public-read \
+	  --exclude "*" \
+	  --include "test-reporter-$(VERSION)-*" \
```

Comment:
Yep, I think that would be a good change to make. I'll add that in a separate PR.